src/main/java/app/bpartners/geojobs/endpoint/event/gen/DetectionJobCreated.java

```diff
@@ -0,0 +1,19 @@
+package app.bpartners.geojobs.endpoint.event.gen;
+
+import app.bpartners.geojobs.repository.model.geo.tiling.TilingTask;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import javax.annotation.processing.Generated;
+import lombok.*;
+
+@Generated("EventBridge")
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@EqualsAndHashCode
+@Data
+@ToString
+public class DetectionJobCreated implements Serializable {
+  @JsonProperty("tilingTask")
+  private TilingTask tilingTask;
```

Comment:
I feel like this is super wrong.

1. Job has multiple tasks. Why just one task here?
2. What is the need to link DetectionJob with TilingTask? I could understand an __optional__ link between DetectionJob and Tiling__JOB__, but I can't understand the need for Tiling__Task__ (and only a single one on top of that).